wordpress-mobile/WordPress-iOS
# Sidebar: Site Picker

Integrate Site Picker and Site Creation flows in the new sidebar.


https://github.com/user-attachments/assets/b3327899-008f-4b62-a2fa-07a943c37e97

I also updated the new self-hosted site flow to be presented modally as other similar flows.

<img width="1316" alt="Screenshot 2024-08-26 at 6 05 27 PM" src="https://github.com/user-attachments/assets/cf2b9f8a-5b1d-4d77-a123-51be111c2df9">


## Known Issues

The sidebar/split view needs to react to the WordPressAuthentificator notifications to reload the sites after creation. It currently doesn't  automatically.

## To test:

- Login with an account that has 4 or more sites
- Verify that you can add sites via the "All Sites" screen and via the "Add Site" button
- Verify that you can change sites using the "All Sites" screen
- (Regressing Testing) Verify the same steps for the existing site picker in the Home page

## Regression Notes
1. Potential unintended areas of impact


2. What I did to test those areas of impact (or what existing automated tests I relied on)


3. What automated tests I added (or what prevented me from doing so)

PR submission checklist:


Testing checklist:
